src/Dapr.Jobs/DaprJobsGrpcClient.cs

```diff
@@ -0,0 +1,216 @@
+
+    // property exposed for testing purposes
+    internal Autogenerated.Dapr.DaprClient Client => client;
+    
+    internal DaprJobsGrpcClient(
+        Autogenerated.Dapr.DaprClient innerClient,
+        HttpClient httpClient,
+        KeyValuePair<string, string>? apiTokenHeader)
+    {
+        this.client = innerClient;
+        this.httpClient = httpClient;
+        this.apiTokenHeader = apiTokenHeader;
+
+        this.httpClient.DefaultRequestHeaders.UserAgent.Add(UserAgent());
+    }
+
+    /// <summary>
+    /// Schedules a job with Dapr.
+    /// </summary>
+    /// <param name="jobName">The name of the job being scheduled.</param>
+    /// <param name="schedule">The schedule defining when the job will be triggered.</param>
+    /// <param name="payload">The main payload of the job.</param>
+    /// <param name="startingFrom">The optional point-in-time from which the job schedule should start.</param>
+    /// <param name="repeats">The optional number of times the job should be triggered.</param>
+    /// <param name="ttl">Represents when the job should expire. If both this and DueTime are set, TTL needs to represent a later point in time.</param>
+    /// <param name="cancellationToken">Cancellation token.</param>
+    [Obsolete("The API is currently not stable as it is in the Alpha stage. This attribute will be removed once it is stable.")]
+    public override async Task ScheduleJobAsync(string jobName, DaprJobSchedule schedule,
+        ReadOnlyMemory<byte>? payload = null, DateTimeOffset? startingFrom = null, int? repeats = null,
+        DateTimeOffset? ttl = null,
+        CancellationToken cancellationToken = default)
+    {
+        ArgumentVerifier.ThrowIfNullOrEmpty(jobName, nameof(jobName));
+        ArgumentVerifier.ThrowIfNull(schedule, nameof(schedule));
+
+        var job = new Autogenerated.Job { Name = jobName, Schedule = schedule.ExpressionValue };
+
+        if (startingFrom is not null)
+            job.DueTime = ((DateTimeOffset)startingFrom).ToString("O");
+
+        if (repeats is not null)
+        {
+            if (repeats < 0)
+                throw new ArgumentOutOfRangeException(nameof(repeats));
+
+            job.Repeats = (uint)repeats;
+        }
+
+        if (payload is not null)
+            job.Data = new Any { Value = ByteString.CopyFrom(payload.Value.Span), TypeUrl = "dapr.io/schedule/jobpayload" };
+
+        if (ttl is not null)
+        {
+            if (ttl <= startingFrom)
+                throw new ArgumentException(
+                    $"When both {nameof(ttl)} and {nameof(startingFrom)} are specified, {nameof(ttl)} must represent a later point in time");
+
+            job.Ttl = ((DateTimeOffset)ttl).ToString("O");
+        }
+
+        var envelope = new Autogenerated.ScheduleJobRequest { Job = job };
+
+        var callOptions = CreateCallOptions(headers: null, cancellationToken);
+
+        try
+        {
+            await client.ScheduleJobAlpha1Async(envelope, callOptions);
```

Comment:
Pass the cancellation token?  And then, if/when a `RpcException` is thrown of type "cancelled" and our token was the requesting one, throw the equivalent `OperationCanceledException` (as we don't want to report that the Dapr endpoint indicated a failure if/when the caller was the one to do the cancellation).